datetime/src/main/java/com/afollestad/materialdialogs/datetime/DatePickerExt.kt

```diff
@@ -46,6 +46,13 @@ fun MaterialDialog.datePicker(
       dialogWrapContent = windowContext.isLandscape()
   )
 
+  check(minDate == null || currentDate == null || minDate.before(currentDate)) {
+    "You `minDate` must be less than `currentDate`."
```

Comment:
Should be "Your" instead of "You", technically my `date-picker` library is already doing this validation as well

Reply:
The only control what `date-picker` have is on `minMaxController` with the following message: "Min date must be less than max date.", but on the last version that control don't exist any more, if I'm not wrong